knowledge-base/how-to-change-the-culture-of-the-datepicker-in-html5-report-viewer.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Change the culture of the calendar in report in HTML5 Report Viewer
+description: This KB article gives a brief description of how to set another language of the Kendo date picker in HTML5 Report Viewer. The approach can be used also for the HTML5 MVC Report Viewer.
```

Comment:
The approach can be used also for other web-based report viewers (because we have WebForms wrapper too).

Reply:
or for all the wrappers of the Html5 Viewer

Reply:
or for all the wrappers of the Html5 Viewer